src/components/home/components/Forms/M2MPanel/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const M2M = () => (
+	<div className="sc-bwzfXH MKmUa">
+		<span>0 LPWAN Nodes</span>
+		<button className="btn-framed-disabled">Connect</button>
+	</div>
+)
+
+export {
+  M2M as M2MPanel
+}
```

Comment:
I would not add something that does not work at all. This will always show 0 Nodes. I think it is just confusing for the users.